src/open_inwoner/utils/tests/playwright.py

```diff
@@ -94,6 +96,19 @@ def setUpClass(cls):
         cls.playwright = sync_playwright().start()
         cls.browser = cls.launch_browser(cls.playwright)
 
+        # Add custom fonts to media folder to avoid test failures
+        fonts = [
+            "heading_font",
+            "body_font_regular",
+            "body_font_bold",
+            "body_font_italic",
+            "body_font_bold_italic",
+        ]
+        for font in fonts:
+            default_storage.save(
+                f"custom_fonts/{font}.ttf", ContentFile(b"", name=f"{font}.ttf")
+            )
```

Comment:
There could be an issue that `default_storage.save()` generates a new name if the file already exists. IIRC @pi-sigma did something for it in the implementation of the feature.

Also maybe @pi-sigma knows if this list of names is defined somewhere in code so this list self-maintains if we change them.

Reply:
I'll reuse `CustomFontName` to get this list of names and I'll look into the duplicate file issue

Reply:
There might also be a fun interaction with the `@temp_media_root()`decorator (although IMHO that should always be active anyway so we could add that).

Reply:
I defined a custom storage (in `open_inwoner.utils.files`) to remove duplicates. That's necessary because otherwise the fille is added with some random string attached to the file name. E.g. if `heading_font` already exists, then adding another yields `heading_font_s40dfg` or something like it, making it impossible to refer to the file.

Reply:
I reused the custom storage and also added the `temp_media_root` decorator